app/commands/donations/payment/create.rb

```diff
@@ -24,7 +24,7 @@ def call
       amount_in_cents: stripe_data.amount
     ).tap do |payment|
       user.update(total_donated_in_cents: user.donation_payments.sum(:amount_in_cents))
-      AwardBadgeJob.perform_later(user, :supporter)
+      Donations::RegisterUserAsDonor.(user, Time.current)
```

Comment:
I was unsure whether to put this command here, with the donations, or to put it with the user commands

Reply:
I'd probably go for `User::RegisterAsDonor` but could go either way.